listenstore/listenstore/listenstore.py

```diff
@@ -31,7 +31,6 @@ def __init__(self, conf):
     def max_id(self):
         return int(self.MAX_FUTURE_SECONDS + calendar.timegm(time.gmtime()))
 
-
```

Comment:
!
Regardless of if this change was needed or not, it's also a good idea to consider the context of each commit you are making. If you make a change to an html template to change timezone layout, an unrelated change to listenstore doesn't make a lot of sense.
If you were specifically making a change to listenstore to help support the different timezones on the template then this would have been an OK change to make.